tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1341,24 +1341,23 @@ def _sub_fit_memory_save_wrapper(
         """Tries to run the sub-fit in a subprocess (managed by ray). Similar to AutoGluon's parallel fold fitting strategies,
         this code does not shut down ray after usage. Otherwise, we would also kill outer-scope ray usage."""
         memory_safe_fits = ds_fit_kwargs.get("memory_safe_fits", True)
-        force_ray_logging = ds_fit_kwargs.get("force_ray_logging", False)
+        enable_ray_logging = ds_fit_kwargs.get("enable_ray_logging", True)
         normal_fit = False
         if memory_safe_fits:
             try:
                 _ds_ray = try_import_ray()
                 if not _ds_ray.is_initialized():
-                    if force_ray_logging:
+                    if enable_ray_logging:
                         logger.info(
                             f"\tRunning DyStack sub-fit in a ray process to avoid memory leakage. "
-                            "Force enabling ray logging (force_ray_logging=True). "
-                            "This may lead to unwanted logging post-DyStack due to a limitation in ray."
+                            "Enabling ray logging (enable_ray_logging=True). Specify `ds_args={'enable_ray_logging': False}` if you experience logging issues."
                         )
-                        _ds_ray.init()  # TODO: This will propagate to the main fit call too, which isn't ideal.
+                        _ds_ray.init()
                     else:
                         logger.info(
                             f"\tRunning DyStack sub-fit in a ray process to avoid memory leakage. "
-                            "Logs will not be shown until this process is complete, due to a limitation in ray. "
-                            "You can force logging by specifying `ds_args={'force_ray_logging': True}`."
+                            "Logs will not be shown until this process is complete (enable_ray_logging=False). "
```

Comment:
I'm a bit confused by the statement, will logs show with `enable_ray_logging=False`

Reply:
The ray process logs will not show if `enable_ray_logging=False`. This log will show because it isn't tied to ray.

Reply:
An example of what the ray logs look like: http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-4262/4de48f7/tutorials/tabular/tabular-essentials.html#maximizing-predictive-performance

Reply:
That is what the next line in the logging message is:

```
"You can experimentally enable logging by specifying `ds_args={'enable_ray_logging': True}`."
```

Reply:
Oh cool!